js/src/lib/components/InferenceWidget/shared/WidgetInputSamplesGroup/WidgetInputSamplesGroup.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+	import { slide } from "svelte/transition";
+	import IconCaretDownV2 from "../../../Icons/IconCaretDownV2.svelte";
+
+	export let classNames = "";
+	export let isLoading = false;
+	export let inputGroups: string[];
+	export let selectedInputGroup: string;
+
+	let containerEl: HTMLElement;
+	let isOptionsVisible = false;
+	let title = "Groups";
```

Comment:
is this string customizable by the model card?

Reply:
At the moment. I can make it so

Reply:
for BLOOM might make sense to name it Language maybe – but not sure how to do this from the schema of `widgetData`....